front/src/applications/operationalStudies/views/Scenario/components/ManageTimetableItem/ManageTimetableItemLeftPanel.tsx

```diff
@@ -179,23 +179,25 @@ const ManageTimetableItemLeftPanel = ({
           </>
         )}
       </div>
-      <div
-        className="scenario-timetable-manage-timetable-item-body"
-        role="button"
-        tabIndex={0}
-        onClick={() => setItineraryModalIsOpen(true)}
-      >
-        <button
-          className="btn btn-light btn-block text-truncate pr-2"
-          title={t('itineraryModal.openItineraryModal')}
-          type="button"
+      {displayTimetableItemManagement !== MANAGE_TIMETABLE_ITEM_TYPES.import && (
```

Comment:
What do you think about using an explicit condition rather than exclusion:

```suggestion
      {(displayTimetableItemManagement === MANAGE_TIMETABLE_ITEM_TYPES.add ||
        displayTimetableItemManagement === MANAGE_TIMETABLE_ITEM_TYPES.edit) && (
```

I think it's : 
- more clear:  we can see exactly when the button shows up
- more safe : if we add a new type later, we need to think about whether to include it

Reply:
`MANAGE_TIMETABLE_ITEM_TYPES` will soon be removed since everything will be done in the same interface (except itinerary). But you are right it might be easier to understand by listing the 2 possibilities :)

Reply:
Done